modules/configuring-a-proxy-after-installation-cli.adoc

```diff
@@ -0,0 +1,80 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring-cluster-wide-proxy.adoc
+
+:_content-type: PROCEDURE
+[id="configuring-a-proxy-after-installation-cli_{context}"]
+= Configuring a proxy after installation using the CLI
+
+You can use the {product-title} (ROSA) CLI (`rosa`) to add a cluster-wide proxy configuration to an existing ROSA cluster in a Virtual Private Cloud (VPC).
+
+You can also use `rosa` to update an existing cluster-wide proxy configuration. For example, you might need to update the network address for the proxy or replace the additional trust bundle if any of the certificate authorities for the proxy expire.
+
+[IMPORTANT]
+====
+The cluster applies the proxy configuration to the control plane and compute nodes. While applying the configuration, each cluster node is temporarily placed in an unschedulable state and drained of its workloads. Each node is restarted as part of the process.
+====
+
+.Prerequisites
+
+* You have installed and configured the latest ROSA (`rosa`) and OpenShift (`oc`) CLIs on your installation host.
+* You have a ROSA cluster that is deployed in a VPC.
+
+.Procedure
+
+* Edit the cluster configuration to add or update the cluster-wide proxy details:
++
+[source,terminal]
+----
+$ rosa edit cluster \
+ --cluster $CLUSTER_NAME \
+ --additional-trust-bundle-file $CA_BUNDLE_FILE \
+ --http-proxy $HTTP_PROXY \
+ --https-proxy $HTTPS_PROXY
+----
++
+While the `additional-trust-bundle-file`, `http-proxy`, and `https-proxy` arguments are optional, if you set a `additional-trust-bundle-file` without either an `http-proxy` or `https-proxy` argument, then the additional trust bundle will still be used for verifying cluster system egress traffic.
```

Comment:
@mrbarge would you help to confirm if  this sentence makes sense

Reply:
This sentence has bugged me a bit, it's a little confusing in the point it's trying to make.

Breaking it up might make it read a bit better.

```suggestion
The `additional-trust-bundle-file` argument can be supplied without needing an accompanying `http-proxy` or `https-proxy` argument. In that scenario, it will still be added to the system's trust store and be used for verifying cluster system egress traffic.
```

Reply:
Thank you both for highlighting this.

A similar suggestion about breaking this sentence up was made by @michaelryanpeter in https://github.com/openshift/openshift-docs/pull/47385#discussion_r924716223. I resolved that in my local feature branch and forgot to push the changes to my fork.

On further review, I noticed that this section is similar to the code block seen in modules/configuring-a-proxy-during-installation-cli.adoc (both sections are being moved in this PR, not created from scratch. That other section provides detailed call outs that are more explicit. I have updated the code block in this module (modules/configuring-a-proxy-after-installation-cli.adoc) so that it also uses call outs and explicitly describes each of the relevant points. I have also included some of the details in @mrbarge's suggestion in the call outs. Additionally, I made sure that the code block and call out in both modules are consistent in their use of variables, etc.

Please let me know if the update looks good or if you have further feedback. Thanks!